### DISCUSSION TITLE
@pmgducati PCB enquiry

### DISCUSSION THREAD

**Discussion by caronova** [darthcloud/BlueRetro#246] [Q&A] [answered]
@pmgducati you are a legend!

I am ordering majority of the boards currently available, and cannot wait to assemble them!

Just wondering when we can expect boards for: NeoGeo/Supergun, Genesis/Megadrive, SNES and Dreamcast. Very keen for these, any news would be great!

I am also wondering we can expect housings for the units, hope the test fits were good. Would each board have its own unique housing?

Thanks again for all your work, please let me know if there is a way to give yourself a bit of a kickback (you too @darthcloud), would really like to for all your effort!

EDIT: Apologies, should have put this post within the pmgducati discussion board itself...

- C

**Comment by pmgducati** [comment, marked as answer]
Hi,
Glad to hear that you are using my designs! I have been working on this for my own collection and was not planning on making any other PCBs.  But in the last couple weeks I have had various requests so it looks like its time to work on some of the stragglers that I left out.  I would be happy to start working the the PCBs above, although I don't have a NeoGeo/Supergun or an SNES so I cant verify the designs.  With that said the only issue that I have is the investment I would need to make to assemble the PCBs that I personally would not use.  The largest cost per PCB is the extension cables for each system, a system side PCB usually runs around $30 each all said and done (PCB,BOM,shipping charges). 

As for the housings, they are in progress (for the existing systems).  You can see them in my repo under the Housing CAD branch.  I have not merged them yet as I made some revisions and waiting on the prints from shapeways (ETA mid April).  If I expanded there would be a housing version with the Logo/Name and correct qty of holes for the controller wires. Here is an example if the 1st rev housing:
![PXL_20220402_015516131](https://user-images.githubusercontent.com/20894227/161899984-91453b01-cfd7-4893-8dbe-4aeb6746bfb1.jpg)
![PXL_20220402_015522374](https://user-images.githubusercontent.com/20894227/161899988-626fb6c9-6fe0-4ddf-9601-5416883229fb.jpg)

If you would like chat feel free to reach out in my repo or reach out to my username at gmail

**Reply by darthcloud** [reply]
Awesome looking shell!

**Comment by pceslayer** [comment]
Oooh! The housing looks quite nice! Can't wait for you to release the .stl
files. I have a few 3D printers if you ever need something fit tested.

If you take the time to design the pcb's I'm sure some of us in the
community that have the consoles could order and test them for you.

**Reply by pmgducati** [reply]
Well, I just finished NeoGeo (Parallel 1P both PP and OD), Atari 2600 (Parallel 2P both PP and OD), Sega Master System (Parallel 2P both PP and OD), Genesis (No Multitap), SNES, CD-i, Jaguar, PC-FX and Dreamcast.  Nothing has been merged yet but they are all in their own branches.  I have a second round of PCB's coming that have some changes related to fitment within the housing.  So until they get here and I can test fit everything I am considering all the new designs pending.

The only systems left are Atari 7800 and JVC which have unique layouts, I will get to them later.

Once I know all is good with the latest round of housings and PCBs I will merge everything and start working on the new housing variants.  (ETA for the next round is end of next week).
